src/main/webapp/team.html

```diff
@@ -1,30 +1,244 @@
+                <div class="p-3 headers">
+                    <div class="row">
+                        <div class="col-2">
+                            <div class="username">
+                                <a href="/">
+                                    <button type="button" class="btn btn-danger" id="homeButton">
+                                        LOG-IN PAGE
+                                    </button>
+                                </a>
+                            </div>
+                        </div>
+                        <div class="col-8">
+                            <h3>MY TASKS</h3>
+                        </div>
+
+                        <!-- Button trigger modal CREATE NEW TASK-->
+                        <div class="col-2 text-start p-1">
+                            <div class="row">
+                                <div class="col">
+
+                                </div>
+                                <div class="col">
+
+                                </div>
+                                <div class="col">
+                                    <button type="button" title="CLICK TO CREATE A NEW TASK" class="btn btn-warning p-1" data-bs-toggle="modal"
+                                        data-bs-target="#createtask" id="createTaskBtn">
+                                        <i class="bi bi-clipboard2-plus"></i>
+                                    </button>
+                                </div>
+                                <div class="col">
+                                        <form name="comTask" action="/complete-task" method="POST" onsubmit="addUserNForm()">
+                                            <input type="hidden" name="userN" class="TaskUserName" type="text"/>
+                                            <button type="submit" title="CLICK TO END CURRENT TASK" class="text-black btn btn-success p-1" id="endTaskBtn" >
+                                            <i class="bi bi-clipboard2-check"></i>
+                                        </form>
+                                    </button>
+                                </div>
+                            </div>
+                            
+                        </div>
+                    </div>
+
+                </div>
+                <div class="mytasks" id="mytasks">
+                    <div class="curtask" id="curtask">
+                        <p class="p" id="currentTasks"></p>
+                    </div>
+
+                    <div class="logboxuser">
+                        <div class="pasttasks" id="pasttasks">
+                            <p class="p" id="pasttask"></p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+
+
+
+            <div class="col-sm-12 col-md-6 col-lg-6 colur ">
+                <div class="p-3 headers">
+                    <h3>MY TEAM ACTIVITY</h3>
+                </div>
+
+                <!-- HARD CODED TASK CONTAINER
```

Comment:
Should we remove this commented out part?

Reply:
Yes... Done